backend/routes/performance_routes.py

```diff
@@ -84,6 +87,21 @@ def get_adjudications(performance_id):
 def create_adjudication(performance_id):
     adjudication_json = request.get_json()
     adjudication_json['performance_id'] = performance_id
+    nomination_comments = adjudication_json['nomination_comment']
```

Comment:
I feel like the key should be plural (i.e. `adjudication_json['nomination_comments']`). what do u think?